### PR TITLE
add changes through pr

### PR DESCRIPTION
**Pull request by victorioso-daniel-2**
fix the loop on victorioso's menu

### CODE REVIEW

File: package/victorioso.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+         goals()
+      elif user_choice == "3":
+         gerald_comment()
+      elif user_choice == "4":
+         michael_comment()
+      elif user_choice == "5": 
+         ivan_comment()
+      elif user_choice == "6":
+         simone_comment()
+      elif user_choice == "7":
```
Comment (delumen-ivan) [inline package/victorioso.py:49]: i will fix this line of code

**Review by delumen-ivan — CHANGES_REQUESTED** [review]

File: package/victorioso.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+def simone_comment():
+   print("")
+
+def menu_function():
+   while True:
+      display_menu()
+
+      user_choice = input("Enter your user_choice: ")
+
+      if user_choice:
```
Comment (delumen-ivan) [inline package/victorioso.py:37]: add = "1" to this line of code

**Review by delumen-ivan — CHANGES_REQUESTED** [review]

File: package/victorioso.py (hunk 2 above)
Comment (victorioso-daniel-2) [reply package/victorioso.py:37]: done

**Review by delumen-ivan — APPROVED** [review]
LGTM ^__^

**Comment by mamasalanang-gerald** [comment]
looks good to me

**Review by mamasalanang-gerald — COMMENTED** [review]
look goods to me

**Pull request merged**